lib/zenaton/client.rb

```diff
@@ -60,6 +62,7 @@ def self.init(app_id, api_token, app_env)
 
     # @private
     def initialize
+      @graphql = Services::GraphQL.new
```

Comment:
In here we could do something like this
```suggestion
      @graphql = Services::GraphQL.new(http: @http)
```
Which I think should allow you do avoid copying much of the http client into the graphql module.